components/clientComponents/globals/FormCaptcha/FormCaptcha.tsx

```diff
@@ -1,50 +1,42 @@
 import HCaptcha from "@hcaptcha/react-hcaptcha";
 import { logMessage } from "@lib/logger";
 import { useFeatureFlags } from "@lib/hooks/useFeatureFlags";
-import { hCaptchaEnabled } from "./helpers";
 import { FormEvent, useRef } from "react";
 
 /**
- * Acts as a HCaptcha wrapper to simplify the wiring around adding hCaptcha to a form. When hCaptcha
- * is enable a form is wired up to use hCaptcha and returned. Otherwise a default form is returned.
- *
- * Note: hCaptcha is currently running in "100% Passive" mode that will never challenge users. The
- * mode is set at account level.
+ * Acts as a hCaptcha wrapper to help simplify the wiring around adding hCaptcha to a form.
  */
 export const FormCaptcha = ({
   children,
-  hCaptchaSiteKey = "",
   handleSubmit,
   lang,
-  id = "",
   dataTestId = "",
-  className = "",
-  isPreview = false,
-  noValidate = true,
+  isPublished = true,
   captchaToken,
+  ...rest
 }: {
   children: React.ReactNode;
-  hCaptchaSiteKey: string | undefined;
-  // Determines whether or not to block a user marked as a bot by siteverify from submitting a form
-  blockableMode?: boolean;
   handleSubmit: (e?: FormEvent<HTMLFormElement>) => void;
   lang: string;
-  id?: string;
   dataTestId?: string;
-  className?: string;
-  isPreview?: boolean;
-  noValidate?: boolean;
+  isPublished?: boolean;
   captchaToken: React.RefObject<string> | undefined;
-}) => {
+} & React.FormHTMLAttributes<HTMLFormElement>) => {
   const hCaptchaRef = useRef<HCaptcha>(null);
   const formSubmitEventRef = useRef<FormEvent<HTMLFormElement>>(null);
 
+  // Help developers understand when there is a configuration issue
   const { getFlag } = useFeatureFlags();
-  const captchaEnabled = hCaptchaEnabled(getFlag("hCaptcha"), hCaptchaSiteKey, isPreview);
+  const hCaptcha = getFlag("hCaptcha");
 
-  if (captchaEnabled && !hCaptchaSiteKey) {
-    logMessage.error("hCaptcha: hCaptchaSiteKey is missing");
-    return null;
+  if (
+    process.env.NODE_ENV === "development" &&
+    hCaptcha &&
+    !process.env.NEXT_PUBLIC_HCAPTCHA_SITE_KEY
+  ) {
+    logMessage.info(`hCaptcha: flag is enabled but hCaptchaSiteKey is missing. This will cause 
```

Comment:
We might want to change this to a `warn` level so it's easier to see among all the other info and debug outputs in the console.